daemon/authorization/get_album_song.py

```diff
@@ -0,0 +1,43 @@
+import urllib.request
```

Comment:
Please change to use pycurl since the pycurl is implemented by C code. The speed is more faster than request.